examples/large_models/Huggingface_pippy/Readme.md

```diff
@@ -0,0 +1,88 @@
+# Loading large Huggingface models with PiPPy (PyTorch Native Large inference solution)
+
+This document briefs on serving large HF model with PiPPy.
+
+PiPPy provides pipeline paralleism for serving large models that woul not fit into one gpu. It takes your model and splits it into equal sizes (stages) partitioned over the number devices you specify. Then uses micro batching to run your batched input for inference ( its is more optimal for batch sizes >1). Microbatching is the techniques in pipeline parallelsim to maximize gpu utiliztion.
+
+## How to serve your large HuggingFace models with PiPPY in Torchserve?
+
+We use a Torchserve custom handler that inherits from base_pippy_handler to load the model and define our logic for preprocess, inference and post processing. This is basically very similar to your evaluation process.
+
+### Step 0: Install torchserve from src
+```bash
+python ts_scripts/install_from_src.py
```

Comment:
So this will be merged in time for the next release right? Can skip from source installation instructions

Also what happens to requirements.txt now, are we planning on adding a seperate CI machine with seperate optional dependencies with seperate tests for all these libraries?